mocca-client/src/main/java/com/paypal/mocca/client/MoccaExceptionHandler.java

```diff
@@ -0,0 +1,33 @@
+package com.paypal.mocca.client;
+
+import feign.FeignException;
+
+import javax.validation.ConstraintViolationException;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * This exception handler allows Mocca to hide Feign exception types from applications
+ *
+ * @author facarvalho
+ */
+class MoccaExceptionHandler<T, R> {
```

Comment:
should be marked final with private constructor.

Reply:
Done